Game/gameState.py

```diff
@@ -43,6 +44,8 @@ def draw(self, window: Surface):
             for e in self.entities:
                 if isinstance(e, Actor):
                     e.drawHealthBar(window)
+            for a in self.animations:
+                a.draw(window)
```

Comment:
This could be moved before the `drawAboveEntities` call, so explosions cannot be seen behind walls